### ISSUE TITLE
SQL Error after adding Monetisation to Apigee Drupal 8 kickstart developer portal

### ISSUE THREAD

**Issue by sushah** [apigee/apigee-m10n-drupal#231]
**Describe the bug**
After adding monetisation extension when I click on 'Pricing & plans' menu, I get the following message 'The website encountered an unexpected error. Please try again later' 

**To Reproduce**
1. Add monetisation extension in Drupal 8
2. Visit your developer portal
3. Click on Pricing & plans

**Expected behavior**
I expected to see the page contents.

**Screenshots**

**Additional context**
'Recent Log messages' under 'reports' admin menu shows:

Drupal\Core\Entity\EntityStorageException: SQLSTATE[HY000]: General error: 1270 Illegal mix of collations (ascii_general_ci,IMPLICIT), (utf8mb4_general_ci,COERCIBLE), (utf8mb4_general_ci,COERCIBLE) for operation ' IN ': SELECT tag, invalidations FROM {cachetags} WHERE tag IN ( :tags__0, :tags__1 ); Array ( [:tags__0] => rate_plan:payment_method_-_credit_card_payment_method_-_credit_card_–_vanilla_rp [:tags__1] => rate_plan:payment_method_-_credit_card_payment_method_-_credit_card_–_vanilla_rp:values ) in Drupal\apigee_edge\Entity\Storage\EdgeEntityStorageBase->withController() (line 220 of /srv/bindings/002185bf61f34e9ba6096f6fd9c9db39/code/web/modules/contrib/apigee_edge/src/Entity/Storage/EdgeEntityStorageBase.php).

**Version Info**
This can be the version you can see on admin/modules in Drupal or 
the output of this command: `composer show`.  Add Drupal core and
other version information if needed.

Admin Toolbar 8.x-2.2
Adminimal Admin Toolbar 8.x-1.10
Apigee API Catalog 8.x-1.1
Apigee Edge 8.x-1.8
Apigee Monetization 8.x-1.4
Better Exposed Filters 8.x-3.0-alpha6
Chaos Tool Suite (ctools) 8.x-3.4
Components! 8.x-1.1
Default Content for D8 8.x-1.0-alpha8
Email Registration 8.x-1.0
Entity API 8.x-1.0
Entity Reference Revisions 8.x-1.8
File Link 8.x-1.7
Font Awesome Icons 8.x-2.15
Key 8.x-1.12
Paragraphs 8.x-1.11
Pathauto 8.x-1.6
Requirement 8.x-1.0
Token 8.x-1.6
Radix 8.x-4.8

** It is very likely I have missed something from the steps but can't see where I have gone wrong. Any suggestions how I can fix this please **

**Comment by cnovak** [comment]
From the [docs about the database setup for Drupal](https://www.drupal.org/docs/8/install/step-3-create-a-database#create-a-database-using-mysqlmariadb-commands) it says:
> The database should be created with UTF-8 (Unicode) encoding (utf8mb4) and either utf8mb4_unicode_ci or utf8mb4_general_ci collation

From the output you have, it looks like one of the tables is using `ascii_general_ci`. It looks like you should make sure the default collation is correct for your database, and that all tables are converted to utf8mb4_general_ci or utf8mb4_unicode_ci.

**Comment by sushah** [comment]
Thanks for the reply. The message is clear on the issue. However the database setup is a black box as neither of the document points to me having to set it up. In the first instance, I followed the instructions [here](https://github.com/stratus-meridian/apigee-kickstart-drupal8-drops) to create a kickstart developer portal on a Pantheon environment. This worked fine. I then added monetisation by following the instructions [here](https://github.com/apigee/apigee-m10n-drupal). This is when I started getting the failure.  I am wondering if there is a pre-requisite step required and does this need to be mentioned in the document.
